app/js/classAdd.controller.js

```diff
@@ -1,10 +1,13 @@
 angular
     .module('conceptDictionaryApp')
-    .controller('ClassAddController', ['$location', 'openmrsRest',
-    function($location, openmrsRest){
+    .controller('ClassAddController', ['$location', 'openmrsRest', '$translate', '$translatePartialLoader',
+    function($location, openmrsRest, $translate, $translatePartialLoader) {
 
         var vm = this;
 
+        $translatePartialLoader.addPart('class/class-add');
+        $translate.refresh();
```

Comment:
Offhand it seems like it will be annoying to have to add both $translate and $translatePartialLoader to every controller. Can't $translatePartialLoader require $translate transitively, and take care of calling refresh itself?